clusterman/kubernetes/kubernetes_cluster_connector.py

```diff
@@ -197,11 +197,18 @@ def get_resource_excluded(self, resource_name: str) -> float:
         )
 
     def get_unschedulable_pods(
-        self,
+        self, detect_reason: Optional[bool] = True
```

Comment:
suggestion: might be nice to be consistent with the function name and name this parameter `get_reason` 

that said, maybe we could even name this something like `get_detailed_reason`? or add a docstring to this function? as it is right now, this is a bit confusing to read since the function signature says that this will return `List[Tuple[KubernetesPod, PodUnschedulableReason]]`, but that's a bit at odds with the added parameter

Reply:
Yeap, I agree with your point. I will delete PodUnschedulableReason and all related thing after migration (after enabling flag everywhere).